integrations/destroy.go

```diff
@@ -0,0 +1,46 @@
+package integrations
+
+import (
+	"fmt"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/utils"
+)
+
+func Destroy(integration string) error {
+	var id string
+	var name string
+
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	if !utils.IsUUID(integration) {
```

Comment:
Usually this kind of check is done on the API side with a method `find_by_id_or_uuid` and not in the CLI. We prefer to have a dumb CLI

Reply:
It is possible to use the UUID or the name of the integration (uuid and scm_type fields in the API side). It should be a method like this `find_by_uuid_or_scm_type` in the API to make this.

Reply:
Do I make this change in the API side?

Reply:
I don't really understand your comment. But yes, I prefer if it's done on the API side